ert/data/_record.py

```diff
@@ -271,22 +305,24 @@ def transmitter_type(self) -> RecordTransmitterType:
 
     async def _transmit(self, record: Record) -> None:
         storage_uri = self._storage_path / self._concrete_key
-        if record.record_type != RecordType.LIST_BYTES:
+        if isinstance(record, NumericalRecord):
```

Comment:
We don't have to do it in this issue, but I think it would be natural with a follow-up issue where we discuss whether it would be possible with a more strict separation between the concrete records and the transmitters. In particular, it the records now can get responsibility for encoding, while the transmitter gets responsibility for how to transmit the encoding that would be great! The gold standard would be that one could introduce a new record type without that impacting the transmitter 🏅

Reply:
I guess that is what I said above, I see now :smile:, I fully agree.

Reply:
Perhaps you could write up an issue on this? :)

Reply:
Sure, will do.